lib/ttnt/test_selector.rb

```diff
@@ -50,6 +42,33 @@ def select_tests
 
     private
 
+    def test_to_code_mapping
```

Comment:
Since you are maintaining a `@mapping` instance variable, to be consistent and because we know which kind of mapping we are maintaining, you could simply rename it `mapping`. What do you think ?